### ISSUE TITLE
Game randomly freezes, but is still "playable" even though nothing moves on the screen

### ISSUE THREAD

**Issue by Viscelius** [LostArtefacts/TRX#686]
Hello.
After installing Tomb1Main, my game freezes everytime I try to play. It's kind of awkward, actually; everything freezes for like 3-4 seconds (the music is still playing, but the action is stopped, as if the game was loading some really heavy content), then after this, even if everything on screen is still frozen, I can hear the game reacting to my inputs; I can hear Lara jump, draw her guns, shoot, the background music is still playing, if Lara was in the middle of a jump when the game froze I can hear her landing after the 3-4 seconds delay, etc... but nothing moves on the screen.

Also, it occurs pretty randomly, sometimes immediately after starting/loading a game, sometimes after 2 minutes of gameplay, sometimes it even happens on the main menu.

Since I had the exact same problem after I installed the TR1 Automated Fix, my guess is that this problem has something to do with TombATI (or with whatever thing both the Automated Fix and Tomb1Main have in common).

I tried reading the log after my last freeze and this is what the last lines were:
![image](https://user-images.githubusercontent.com/121802922/210251002-1deaba29-c5af-43ad-b40d-d2bf52644282.png)

I don't know if it's worth mentioning, but I can't open "Tomb1Main_ConfigTool" and "Tomb1Main-2.12-Installer"on my computer, so I had to install everything manually. I allowed both of them through Windows firewall, Windows Defender detects no threat when I scan them (and for now, I have no other antivirus software installed), I tried to run them as administrator without success, so I really, really don't know why nothing happens when I double click on them. The computer seems to react, as the small hourglass appears next to my cursor, but it quickly disappears and nothing happens. 
I tried to download the installer again; when I try to run it, Windows displays a warning message and asks me if I really want to run the executable as it seems dangerous. Even if I answer yes, nothing happens. When I try to run the executable again, no warning message, but still, nothing works.

I'm getting a bit desperate, as I have absolutely no idea what to do to solve this. I tried to look online for answers, but the ONLY person I found who seems to have the same problem as me is some Steam user who complained recently on the Automated Fix page, saying "this patch destroyed my entire game, game freezes after 10 sec every time".

I have Windows 10 64x. I'm using a powerful recent laptop so I don't think it's a problem related to the hardware I'm using, thus I don't know if writing down all the components of my computer would be relevant.

I'm trully sorry if it's the wrong place to ask for help, but I don't know where I could ask except here.
Thank you for reading this.

**Comment by walkawayy** [comment]
Hm can you post what CPU and GPU your laptop have just for reference.

I haven't heard of any issue like this before. Can you also toggle the FPS key (default is F2) and report back what that shows during gameplay?

**Comment by Viscelius** [comment]
Thanks for your answer ! Actually I've just solved the problem, thanks to you. 

I was checking what my GPU exactly is, and noticed that my laptop has two built in GPUs:
- Intel UHD Graphics
- NVIDIA GeForce RTX 3070 Laptop GPU

I wasn't aware of that. Anyway, I don't know much about hardware, but I suspected my computer was using the wrong GPU when the game was running. 
I opened the Nvidia Control Panel, clicked "Manage 3D settings", and changed the game preferred graphics processor to "High-performance NVIDIA Processor". It solved the issue.

I also tested this solution with my Steam version of the game, with the Automated Fix installed. It also worked. That's the first time I have to manually change the preferred GPU of one of my game, so I didn't think of that when trying to solve the issue.

Anyway, everything's fine now. Thanks !
